python/analyzers/HNLAnalyzer.py

```diff
@@ -90,75 +92,107 @@ def process(self, event):
         
         # store the number of sMu and dSAMu per event
         event.n_sMu = len(event.sMu)
-        # event.maxptsMu=sorted(event.sMu,key=lambda x: x.pt(),reverse = True)[0] 
-
         event.n_dSAMu = len(event.dSAMu)
-        # event.maxptdSAMu=sorted(event.dSAMu,key=lambda x: x.pt(),reverse=True)[0]
-        
-        ##############
-        # for now ONLY work with sMu
-        ##############
+       
+        # Merge Reco Muons
+        # Create an array of DisplacedMuon objects, summarizing all sMu and dSAMus into a single array, while avoiding redundancies through dR<0.2
+        dMus = []
+        dxy_cut = 1000 # cut selection for sMu / dSAMu in mm
```

Comment:
better have it configurable. No big deal for now, just to keep the record

Reply:
What is the most "correct" way to set the tune parameters for an analysis?